NetworkDispatcherKtor/src/androidMain/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="com.sdk.growthbook"
```

Comment:
Could you please clarify why package was removed?

Reply:
This was done to resolve the following warning during build time:
```
> Task :NetworkDispatcherKtor:processDebugManifest
package="com.sdk.growthbook" found in source AndroidManifest.xml: /home/farmerbb/growthbook-kotlin/NetworkDispatcherKtor/src/androidMain/AndroidManifest.xml.
Setting the namespace via a source AndroidManifest.xml's package attribute is deprecated.
Please instead set the namespace (or testNamespace) in the module's build.gradle file, as described here: https://developer.android.com/studio/build/configure-app-module#set-namespace
This migration can be done automatically using the AGP Upgrade Assistant, please refer to https://developer.android.com/studio/build/agp-upgrade-assistant for more information.
```

Having the `package` attribute in the manifest results in an error when using Android Gradle Plugin 8.0.0 or higher. 
 However, it doesn't matter as much now that we've downgraded back to 7.4.2 for this PR.